src/pycram/external_interfaces/robokudo.py

```diff
@@ -1,135 +1,169 @@
 import sys
-from typing_extensions import Callable
+from threading import Lock, RLock
+from typing import Any
 
-import rospy
 import actionlib
 import rosnode
+import rospy
+from geometry_msgs.msg import PointStamped
+from typing_extensions import List, Callable, Optional
 
-from ..designator import ObjectDesignatorDescription
 from ..datastructures.pose import Pose
-from ..local_transformer import LocalTransformer
-from ..datastructures.world import World
-from ..datastructures.enums import ObjectType
+from ..designator import ObjectDesignatorDescription
+
+is_initialized = False
 
 try:
-    from robokudo_msgs.msg import ObjectDesignator as robokudo_ObjetDesignator
+    from robokudo_msgs.msg import ObjectDesignator as robokudo_ObjectDesignator
     from robokudo_msgs.msg import QueryAction, QueryGoal, QueryResult
 except ModuleNotFoundError as e:
-    rospy.logwarn(f"Could not import RoboKudo messages, RoboKudo interface could not be initialized")
+    rospy.logwarn("Failed to import Robokudo messages, the real robot will not be available")
+
+is_init = False
```

Comment:
Whats the difference between that and the is_initialized above?

Reply:
was just bad cop and paste